src/TestCase.php

```diff
@@ -41,10 +34,6 @@ public function setUp()
         if (! $this->app) {
             $this->refreshApplication();
         }
-
-        if (! $this->factory) {
-            $this->factory = $this->app->make(Factory::class);
```

Comment:
Taylor decided that it's fine because this feature is not documented, and laravel itself already made this change.